master/security/icmp-ping.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Use ICMP/ping in policy rules
+---
+
+### Big picture
+
+Use Calico network policy to allow and deny ICMP/ping messages. 
+
+### Value
+
+The **Internet Control Message Protocol (ICMP)** provides valuable network diagnostic functions, but it can also be used maliciously. Attackers can use it to learn about your network, or for DoS attacks. Using Calico network policy, you can control where ICMP is used. For example, you can:
+
+- Allow ICMP ping, but only within your cluster 
+- Allow ICMP for pods launched by operators for diagnostic purposes, but block other uses
+- Temporarily enable ICMP to diagnose a problem, then disable it after the problem is resolved
+- Deny/allow ICMP messages (ICMPv4 and/or ICMPv6)
```

Comment:
Does this mean to say "Deny or allow specific ICMP message types and codes"?

Reply:
Yes, that is the idea. But in Value, we don't know if users know "type and code" but they would know ICMPv4 and v6.  I like the addition of "specific" though. OK?